src/pages/documentation/index.md

```diff
@@ -17,9 +17,9 @@ The latest versions of the Adobe Experience Platform SDK for Google Android and
 
 <InlineAlert variant="success" slots="header, text"/>
 
-Project Griffon will be Assurance!
+Project Griffon is now Assurance
 
-We're pleased to announce that Project Griffon will be generally available to all Adobe Experience Cloud customers as Assurance. To learn more about this transition see [here](./platform-assurance/index.md) details.
+We're pleased to announce that Project Griffon is now generally available to all Adobe Experience Cloud customers as Assurance. To learn more about this transition see [here](./platform-assurance/index.md) details.
```

Comment:
Since we're changing phrasing anyways, might as well change it to "... about this transition, please read the [Assurance overview] for more details."